src/app/static/src/js/utils/types.ts

```diff
@@ -50,3 +50,15 @@ export interface ReportDependencies {
     direction: "upstream" | "downstream",
     dependency_tree: ReportDependency
 }
+
+export interface ReportMetadata {
```

Comment:
It's good to have a type for this, but would be even better to use it in `runReport.vue`